modules/tinymce/src/plugins/paste/main/ts/api/Api.ts

```diff
@@ -7,12 +7,13 @@
 
 import { Clipboard } from './Clipboard';
 
-const get = (clipboard: Clipboard, quirks) => {
```

Comment:
`quirks` return type was `void` so this was always undefined so I've just removed it entirely.